src/backend/tests/brilisp/castop_uitofp.sexp

```diff
@@ -0,0 +1,21 @@
+(brilisp
+    (define ((fprint float) (n float)))
+
+    (define ((dprint double) (n double)))
+
+    (define ((cas_t float) (n int32))
```

Comment:
can you ditch functions in all the tests and directly put everything in tests please?